spec/factories/intakes.rb

```diff
@@ -269,11 +269,13 @@
       primary_consented_to_service_at { 2.weeks.ago }
       completed_at { 1.week.ago }
       after(:build) do |intake|
-        Intake.defined_enums.keys.each_with_object({}) do |key, hash|
-          intake[key] = Intake.send(key.pluralize).keys.sample
+        Intake.defined_enums.each_key do |key|
+          # only randomize values for keys that have not been supplied
+          unless intake[key]
+            intake[key] = Intake.send(key.pluralize).keys.sample
+          end
         end
       end
     end
```

Comment:
Totally appreciate that you did the work to keep this code from breaking. I think we should probably just delete it though. Sarah & I are both inclined to remove randomization from the factories.

Reply:
There are definitely tests that rely on values for each enumerable being created (the csv specs in particular), so maintaining it and making it work for situations that rely on it and those that need to be more specific seemed like the lowest-impact way forward right now.